internal/graph/graph.go

```diff
@@ -179,22 +201,21 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 				},
 			}, nil
 		}
-
+		// else, we recurse on document#writer
```

Comment:
@adriantam do these comments help you read this code? To me they add noise and don't really help, but if they help you then that's majority and we should keep em'